Cargo.lock

```diff
@@ -3967,36 +3975,10 @@ dependencies = [
  "tracing",
 ]
 
-[[package]]
-name = "hotshot-qc"
-version = "0.3.3"
-source = "git+https://github.com/EspressoSystems/hotshot?rev=ddb8d6f8d058ea456b2c11241cc67b821d77aeee#ddb8d6f8d058ea456b2c11241cc67b821d77aeee"
-dependencies = [
- "ark-bls12-377",
- "ark-bls12-381",
- "ark-bn254",
- "ark-ec",
- "ark-ff",
- "ark-pallas",
- "ark-poly",
- "ark-serialize 0.4.2",
- "ark-std 0.4.0",
- "bincode",
- "bitvec 1.0.1",
- "ethereum-types 0.14.1",
- "generic-array",
- "hotshot-types",
- "jf-primitives",
- "jf-relation",
- "jf-utils",
- "serde",
- "typenum",
-]
-
 [[package]]
 name = "hotshot-query-service"
 version = "0.0.7"
-source = "git+https://github.com/EspressoSystems/hotshot-query-service?branch=main#41586038033c6b4afa6b9494a5e8741b2e99ccfc"
+source = "git+https://github.com/EspressoSystems/hotshot-query-service?branch=main#bbb1327fa69e68df0f039009ac7e5f15001748d2"
```

Comment:
This commit is not the current tip of `main` for `hotshot-query-service`

Reply:
Sure, I can update